Sources/NIOCore/FileHandle.swift

```diff
@@ -388,7 +387,6 @@ extension NIOFileHandle {
     ///   - path: The path of the file to open. The ownership of the file descriptor is transferred to this `NIOFileHandle` and so it will be closed once `close` is called.
     ///   - mode: Access mode. Default mode is `.read`.
     ///   - flags: Additional POSIX flags.
-    @available(*, noasync, message: "This method may block the calling thread")
```

Comment:
@glbrntt We can keep it on the new methods though, no?

Reply:
Yeah, you're absolutely right.